src/parseTools.js

```diff
@@ -422,15 +398,10 @@ function makeSetValue(ptr, pos, value, type, noNeedFirst, ignore, align, sep = '
     const bytes = Runtime.getNativeTypeSize(type);
     if (needSplitting) {
       let ret = '';
-      if (isIntImplemented(type)) {
```

Comment:
Perhaps assert `isIntImplemented(type)` here? And above, `!(type == 'double' && align < 8)`? Or are those obviously impossible somehow?

Reply:
I think they must be impossible because if either of them were true it would have led to tempDoublePtr being referenced, and that pointer has not existed for a long time (not since fastcomp).